app/views/test.scala.html

```diff
@@ -0,0 +1,2347 @@
+@import assets.AssetsResolver
+@import views.ViewHelpers._
+@import admin.AllSettings
+
+@(
+  title: String,
+  mainId: String,
+  mainJsBundle: String,
+  mainStyleBundle: String,
+  description: Option[String] = None,
+  canonicalLink: Option[String] = None,
+  hrefLangLinks: Map[String, String] = Map(),
+  styles: Html = Html(""),
+  scripts: Html = Html(""),
+  data: Map[String, String] = Map.empty,
+  csrf: Option[String] = None,
+  annualAmounts: (String, String, String, String)
+)(implicit assets: AssetsResolver, request: RequestHeader, settings: AllSettings)
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="google-site-verification" content="nFrhJ3suC2OpKRasEkZyH1KZKpSZc-ofno_uunJgfvg" />
+
+    @csrf.map { token =>
+      <script type="text/javascript">
+      window.guardian = window.guardian || {};
+      if(!window.guardian.csrf) {
+        window.guardian.csrf = {token: "@token"}
+      }
+    </script>
+    }
+
+    @description.map { definedDescription =>
+      <meta name="description" content="@definedDescription" />
+      <meta property="og:description" content="@definedDescription" />
+    }
+
+    @canonicalLink.map { definedCanonicalLink =>
+      <link rel="canonical" href="@definedCanonicalLink" />
+    }
+
+    @for((lang, link) <- hrefLangLinks){
+      <link rel="alternate" href="@link" hreflang="@lang" />
+    }
+
+    @settingsScript(settings)
+
+    <title>@title</title>
+    <meta property="og:title" content="@title"/>
+
+    @optimizeScript()
+    <style>
```

Comment:
could we inline the minified CSS which is delivered to the browser in prod instead? this will be larger than what gets sent as a separate file normally, which seems a wasted opportunity